starknet_devnet/blocks.py

```diff
@@ -306,3 +306,22 @@ async def store_pending(
     def get_state(self, block_number: int) -> StarknetState:
         """Return state at block with `number`"""
         return self.__state_archive.get(block_number)
+
+    async def abort_block_by_hash(self, block_hash: str) -> str:
+        """
+        Abort block by given block hash.
+        """
+        numeric_hash = _parse_block_hash(block_hash)
+
+        if numeric_hash in self.__hash2num:
+            block_number = self.__hash2num[int(block_hash, 16)]
+            block = await self.get_by_number(block_number)
+
+            block_dict = block.dump()
```

Comment:
Add a comment here that this is done as it is because block object's properties cannot be modified.

Reply:
fixed in 86a284cbc37ec589309d16f26d570557f634fa7e